### ISSUE TITLE
Attempt to invoke virtual method 'android.app.Application android.app.Activity.getApplication()' on a null object reference

### ISSUE THREAD

**Issue by alexsegura** [Countly/countly-sdk-react-native-bridge#28]
Hi. 

I have this error in Crashlytics, and some users are reporting that the app is crashing. 
It happens _sometimes_, on various versions of Android (7, 8, 9...)

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'android.app.Application android.app.Activity.getApplication()' on a null object reference
       at ly.count.android.sdk.react.CountlyReactNative.init(CountlyReactNative.java:123)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
       at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:151)
       at com.facebook.react.bridge.queue.NativeRunnable.run(NativeRunnable.java)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:27)
       at android.os.Looper.loop(Looper.java:193)
       at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:226)
       at java.lang.Thread.run(Thread.java:764)
```

I tried Pokemon programming but it doesn't help ([see the code](https://github.com/coopcycle/coopcycle-app/blob/58030740128a3b0ed72754ee409e5c197ea2a4e5/src/analytics/CountlyTracker.js)).

```javascript
try {

  Countly.init(COUNTLY_SERVER_URL, COUNTLY_APP_KEY, deviceId)
    .then(() => {
      Countly.start();
    })
    .catch(e => console.log(e));

} catch (e) {
  console.log(e)
}
```

**Comment by ijunaid** [comment]
In React Native [sometimes getCurrentActivity() returns null](https://github.com/facebook/react-native/issues/18345). This is a issue that happens on RN side.
We fixed the issue that causes the crash, we will make a new release later this week with this fix.

**Comment by alexsegura** [comment]
Is it fixed in #29?

**Comment by ijunaid** [comment]
@alexsegura yes, safety checks are added so that exception will not occur in case React Native returns null Activity.

**Comment by alexsegura** [comment]
Fixed in #29 

Thank you @ijunaid 👍
